SiT.Scheduler.Data.PostgreSql/DbContextFactory.cs

```diff
@@ -0,0 +1,21 @@
+﻿using Microsoft.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore.Design;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace SiT.Scheduler.Data.PostgreSql
+{
+    class DbContextFactory : IDesignTimeDbContextFactory<PostgreDbContext>
+    {
+        public PostgreDbContext CreateDbContext(string[] args)
+        {
+            var optionsBuilder = new DbContextOptionsBuilder<PostgreDbContext>();
+            optionsBuilder.UseNpgsql(args[0]);
```

Comment:
We need to validate this and provide additional information about how exactly to prepare the required data.

Reply:
Do i need to add something? I don't understand what i need to do exactly.

Reply:
1. args may be null -> Null reference exception
2. args may be empty -> Index out of range exception
3. args[0] may be null

Reply:
throw new InvalidOperationException("Please, provide a connection string");